examples/ChimneySwap/.gitignore

```diff
@@ -0,0 +1,10 @@
+*.iml
+.gradle
+/local.properties
+/.idea/workspace.xml
+/.idea/libraries
+.DS_Store
+/build
+/captures
+/app/google-services.json
+/app/src/main/res/values/api_keys.xml
```

Comment:
It is file a to put the api keys. I just removed the keys to give people the format for it. The .gitignore keeps it from being uploaded with real values.